### PR TITLE
Add deployment docs

### PR DESCRIPTION
**Pull request by muff1nman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,174 @@
...
+# Requirements
 
 * Alertmanager API v2
 * Kubernetes 1.12+
+
+# Deployment
+1. Download the manifests from the latest Github release
+
+```
+wget https://github.com/cloudflare/sciuro/releases/download/v0.1.0/cluster.yaml
```
Comment (cbroglie) [inline README.md:23]: Can you give an example without hardcoding the version? Might be annoying to keep this updated with new releases

**Review by cbroglie — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (muff1nman) [reply README.md]: Yeah looks like that's possible.

File: README.md [hunk 2]
```diff
@@ -4,13 +4,174 @@
...
+wget https://github.com/cloudflare/sciuro/releases/latest/download/stable.yaml
+```
+
+2. Apply the cluster-scoped resources that allow Sciuro to read nodes and
+   modify their status. If you choose a different namespace, adjust the
+   namespace name and `ClusterRoleBinding` accordingly.
+
+```
+# Review manifests and make adjustments for different namespace
+kubectl apply -f cluster.yaml
```
Comment (terinjokes) [inline README.md:33]: You can kubectl these addresses directly.

```
kubectl apply -f https://github.com/cloudflare/sciuro/releases/latest/download/cluster.yaml
```

Comment (muff1nman) [reply README.md:33]: True, but I think its better to encourage people to read the manifests. In addition the configmap in the `stable.yaml` has to be modified so might as well keep things symmetrical.

**Pull request merged**